packages/core/src/chains.ts

```diff
@@ -1,32 +1,33 @@
-import * as chainRegistry from "chain-registry";
-import * as initiaRegistry from "@initia/initia-registry";
+import { Chain } from '@chain-registry/types';
+import * as chainRegistry from 'chain-registry';
+import * as initiaRegistry from '@initia/initia-registry';
 
 /** @deprecated */
 const DYDX_CHAIN = {
-  $schema: "../chain.schema.json",
-  chain_name: "dydx",
-  status: "live",
-  website: "https://dydx.exchange/",
-  network_type: "mainnet",
-  pretty_name: "dYdX Protocol",
-  chain_id: "dydx-mainnet-1",
-  bech32_prefix: "dydx",
-  daemon_name: "dydxprotocold",
-  node_home: "$HOME/.dydxprotocol",
-  key_algos: ["secp256k1"],
+  $schema: '../chain.schema.json',
+  chain_name: 'dydx',
+  status: 'live',
+  website: 'https://dydx.exchange/',
+  network_type: 'mainnet',
+  pretty_name: 'dYdX Protocol',
+  chain_id: 'dydx-mainnet-1',
+  bech32_prefix: 'dydx',
+  daemon_name: 'dydxprotocold',
+  node_home: '$HOME/.dydxprotocol',
+  key_algos: ['secp256k1'],
```

Comment:
@codingki you have a linter too that's doing all this i assume lol, it's so annoying

Reply:
yeah xd, previously we have a diff eslint config for core and widget. in this monorepo now same